src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+complete a purchase on desktop, but lower mobile conversion rates are also a result of poor user 
+experience. Your job is to *minimize* lost conversions on mobile and *maximize* conversions 
+on desktop. [Research has shown](https://www.comscore.com/Insights/Presentations-and-Whitepapers/2017/Mobiles-Hierarchy-of-Needs) that there's a huge opportunity to provide a better mobile form experience. 
+
+Most of all, users are more likely to abandon forms that look long, complex and without a sense of 
+direction—especially on smaller screens and when users are distracted or in a rush. Ask for as 
+little data as possible.
+
+### Make guest checkout the default {: #guest-checkout}
+
+For an online store, the simplest way to reduce form friction is to make guest checkout the default. 
+Don't force users to create an account before making a purchase. Not allowing guest checkout is cited 
+as a major reason for shopping cart abandonment.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="images/shopping-cart-abandonment.png" alt="Reasons for shopping 
+  cart abandonment during checkout.">
+  <figcaption class="w-figcaption">From <a href="https://baymard.com/checkout-usability">baymard.com/checkout-usability</a></figcaption>
+</figure>
+
+You can offer account sign-up after checkout.  At that point, you already have most of the data you 
+need to set up an account, so account creation should be quick and easy for the user. 
+
+{% Aside 'gotchas' %}
+If you do offer sign-up after checkout, make sure that the purchase the user just made is linked to 
+their newly created account!
+{% endAside %}
+
+### Show checkout progress  {: #checkout-progress}
+
+You can make your checkout process feel less complex by showing progress and making it clear what 
+needs to be done next. The video below shows how UK retailer [johnlewis.com](https://www.johnlewis.com) 
+show checkout progress.
+
+<figure class="w-figure">
+   <video controls autoplay loop muted class="w-screenshot">
+     <source src="https://samdutton.com/johnlewis.com.mp4" type="video/mp4">
+   </video>
+  <figcaption class="w-figcaption">Show checkout progress</figcaption>
+</figure>
+
+You need to maintain momentum! For each step towards checkout, use descriptive buttons that make 
+the next step obvious. For example, label the submit button on your shipping address form 
+**Proceed to Payment** rather than **Continue** or **Save**.
+
+<figure class="w-figure">
+   <video controls autoplay loop muted class="w-screenshot">
+     <source src="https://samdutton.com/address-form-save.mp4" type="video/mp4">
+   </video>
+  <figcaption class="w-figcaption">Give form buttons meaningful names.</figcaption>
+</figure>
+
+Use the `enterkeyhint` attribute on form inputs to set the mobile keyboard enter key label. For 
```

Comment:
TIL!

Reply:
Yeah! All sites should do this. I've even noticed this is wrong in Android settings...